kolena/annotation.py

```diff
@@ -42,15 +43,18 @@ def _data_category() -> str:
         return "ANNOTATION"
 
 
-@dataclass(frozen=True, config=ValidatorConfig)
+@dataclass(frozen=True, config=ValidatorConfig, repr=False)
 class Annotation(TypedDataObject[_AnnotationType], metaclass=ABCMeta):
     """The base class for all annotation types."""
 
     def __init_subclass__(cls, **kwargs):
         _register_data_type(cls)
 
+    def __repr__(self):
```

Comment:
I suggest using `__str__` instead. That is what pandas is using to dump the object into a csv cell.
It convention that `__repr__` outputs valid python code to construct the object

Reply:
https://stackoverflow.com/questions/1436703/what-is-the-difference-between-str-and-repr

Reply:
Nevermind. Containers will use the contained objects __repr__ so we need to use __repr__ here

Reply:
Both repr and str needs to be changed after experimentation